src/app/app.module.ts

```diff
@@ -1,13 +1,30 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-import {NgxPaginationModule} from 'ngx-pagination';
+import { NgxPaginationModule } from 'ngx-pagination';
+import { MatCardModule } from '@angular/material/card';
+import {MatButtonModule} from '@angular/material/button';
+import {MatChipsModule} from '@angular/material/chips';
+import {MatPaginatorModule} from '@angular/material/paginator';
 
 import { AppComponent } from './app.component';
 import { PlacesComponent } from './places/places.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import {MatSelectModule} from '@angular/material/select';
+import {MatGridListModule} from '@angular/material/grid-list';
 
 @NgModule({
   declarations: [AppComponent, PlacesComponent],
-  imports: [BrowserModule, NgxPaginationModule],
+  imports: [
+    BrowserModule,
+    NgxPaginationModule,
+    BrowserAnimationsModule,
+    MatCardModule,
+    MatButtonModule,
+    MatChipsModule,
+    MatPaginatorModule,
+    MatSelectModule,
+    MatGridListModule,
+  ],
```

Comment:
It is just a hint - all material modules should be imported in separate shared module and this shared module should be imported in app.module, for me you are not obligated to change anything it is your choice.